packages/tools/fetch-tool/src/fluidFetchInit.ts

```diff
@@ -137,6 +156,13 @@ export async function fluidFetchInit(urlStr: string) {
         const url = new URL(urlStr);
         const server = url.hostname.toLowerCase();
         return initializeR11s(server, url.pathname, resolvedUrl);
+    } else if (resolvedUrl.type === "fluid") {
```

Comment:
I do not fully follow this code, but "fluid" -> Azure mapping seems fishy. I.e. can't there by any other URI of "fluid" type that is not tied to Azure?

Reply:
Updated to instead check for `fluidrelay.azure.com` in the URL. I tried using protocol to match the other two, but that requires changing the document URL itself which uses `https://` and causes errors in communicating with the service